src/zeep/wsdl/attachments.py

```diff
@@ -75,6 +75,6 @@ def content(self):
         if encoding == 'base64':
             return base64.b64decode(content)
         elif encoding == 'binary':
-            return content
+            return content.strip(b'\r\n')
```

Comment:
Why the strip? Looks strange to me, and potentially wrong, what if the binary content happens to have those bytes at the start and/or end on purpose?